.github/workflows/regression.yml

```diff
@@ -29,10 +29,11 @@ jobs:
   regression:
     runs-on: ubuntu-latest
     strategy:
+      max-parallel: 4
       fail-fast: false
       matrix:
         include:
-          - { tool: klayout, part: drc, test: main }
```

Comment:
we probably want to keep main tested somehow since it perform some basic testing and reporting about the design (maybe by disabling the test that are otherwise covered, ex: dualgate for now)

Reply:
@proppy as I mentioned above `main` will run everything. It's as if you are running dualgate, table1, table2, etc....

We will add table by table moving forward. And actually we will automate getting the tables. `main` is not really useful.

Reply:
can't you selectivly disable dualgate now if you set `FEOL` to false? that way you can still cover everything else to check if there are any syntax / reporting error.

Reply:
I'm not sure what's the question here. Could you elaborate?

Reply:
sorry for not being very clear, the point I'm trying to make here is that we should leave the `main drc` job running here to cover the content of `main.drc` with the regression tests (otherwise we wouldn't catch a PR introduce a typo to `main.drc` for example).

I wouldn't worry about running the same test twice as `matrix` allow us to do this in parallel and we could pass flags/override vars in main.drc to skip some of the deck.

Does that make sense?

Reply:
Main is just the header of all rule decks. It has no rules to test. Also, when you run dualgate for example it takes main and tail and build a concatenated rule deck to run on the deck. Meaning main and tail is always used for testing.

Reply:
ah I see, I guess I miss-understood `main` (I thought it would include all the rules and run them) I didn't realize it was more of a `head`.